controllers/comments.controller.js

```diff
@@ -43,4 +43,14 @@ exports.postComments = (req, res, next) => {
     .catch((err)=>{
         next(err);
     })
+}
+
+exports.deleteComment = (req, res, next) => {
+    return removeComment(req.params.comment_id)
+    .then(() =>{
+        res.status(204).send({message: 'Deleted'});
```

Comment:
Sending a status back would be enough here.
Anything like a message would be better done in the front end.